main/src/main/scala/sbt/Project.scala

```diff
@@ -192,6 +201,15 @@ sealed trait ClasspathDep[PR <: ProjectReference] { def project: PR; def configu
 final case class ResolvedClasspathDependency(project: ProjectRef, configuration: Option[String]) extends ClasspathDep[ProjectRef]
 final case class ClasspathDependency(project: ProjectReference, configuration: Option[String]) extends ClasspathDep[ProjectReference]
 
+/** Indicate whether the project was created organically, or synthesized by a plugin. */
```

Comment:
Could you explain what GenericRoot is (somehow)? Either here or on the case objects.